tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Linker.scala

```diff
@@ -307,6 +297,63 @@ final class Linker(semantics: Semantics, outputMode: OutputMode,
         inheritedMDef.hash) // over-approximation
   }
 
+  private def synthesizeReflectiveProxy(
+      classInfo: Analysis.ClassInfo, methodInfo: Analysis.MethodInfo,
+      targetName: String, getTree: TreeProvider, analysis: Analysis)(
+      implicit pos: Position): MethodDef = {
```

Comment:
I wouldn't position this at the position of the containing class def, but at the position of the method this is calling. It will give much better debugging experience when stepping.